dagshub_annotation_converter/ir/image/annotations/bbox.py

```diff
@@ -0,0 +1,20 @@
+from dagshub_annotation_converter.ir.image.annotations.base import IRAnnotationBase
+
+
+class IRBBoxAnnotation(IRAnnotationBase):
+    top: float
+    left: float
+    width: float
+    height: float
```

Comment:
Definitely missing "confidence" field (this is probably true of the other annotation types as well)